### ISSUE TITLE
haskell-tools-debug build failure with GHC 8.0.2

### ISSUE THREAD

**Issue by snoyberg** [haskell-tools/haskell-tools#257]
```
[3 of 4] Compiling Language.Haskell.Tools.DebugGhcAST ( Language/Haskell/Tools/DebugGhcAST.hs, dist/build/Language/Haskell/Tools/DebugGhcAST.o )

Language/Haskell/Tools/DebugGhcAST.hs:72:1: error:
    • No instance for (Show (HsSplicedThing RdrName))
        arising from a use of ‘showsPrec’
    • In the second argument of ‘(.)’, namely ‘(showsPrec 11 b2)’
      In the second argument of ‘(.)’, namely
        ‘(.) GHC.Show.showSpace (showsPrec 11 b2)’
      In the second argument of ‘(.)’, namely
        ‘(.) (showsPrec 11 b1) ((.) GHC.Show.showSpace (showsPrec 11 b2))’
      When typechecking the code for ‘showsPrec’
        in a derived instance for ‘Show (HsSplice RdrName)’:
        To see the code I am typechecking, use -ddump-deriv

Language/Haskell/Tools/DebugGhcAST.hs:149:1: error:
    • No instance for (Show (HsSplicedThing Name))
        arising from a use of ‘showsPrec’
    • In the second argument of ‘(.)’, namely ‘(showsPrec 11 b2)’
      In the second argument of ‘(.)’, namely
        ‘(.) GHC.Show.showSpace (showsPrec 11 b2)’
      In the second argument of ‘(.)’, namely
        ‘(.) (showsPrec 11 b1) ((.) GHC.Show.showSpace (showsPrec 11 b2))’
      When typechecking the code for ‘showsPrec’
        in a derived instance for ‘Show (HsSplice Name)’:
        To see the code I am typechecking, use -ddump-deriv

Language/Haskell/Tools/DebugGhcAST.hs:226:1: error:
    • No instance for (Show (HsSplicedThing Id))
        arising from a use of ‘showsPrec’
    • In the second argument of ‘(.)’, namely ‘(showsPrec 11 b2)’
      In the second argument of ‘(.)’, namely
        ‘(.) GHC.Show.showSpace (showsPrec 11 b2)’
      In the second argument of ‘(.)’, namely
        ‘(.) (showsPrec 11 b1) ((.) GHC.Show.showSpace (showsPrec 11 b2))’
      When typechecking the code for ‘showsPrec’
        in a derived instance for ‘Show (HsSplice Id)’:
        To see the code I am typechecking, use -ddump-deriv
```

**Comment by nboldi** [comment]
Seems like they removed a typeclass instance from GHC. I will need the new snapshot to fix the problem.

**Comment by nboldi** [comment]
Should be solved in haskell-tools-debug-0.5
